src/components/Common/Button/FbLoginButton/FbLoginButton.tsx

```diff
@@ -21,26 +21,15 @@ export class FacebookLoginComponent extends Component<
   }
 
   handleFacebookResponse = (response: any): void => {
-    const userObj = {
+    const userDetails = {
       firstName: response.first_name,
       lastName: response.last_name,
       email: response.email,
       birthday: response.birthday,
       location: response.location.name
     };
 
-    const authorizeUser = fetch(
-      'http://localhost:8081/api/v1/public/user_authenticate',
-      {
-        method: 'POST',
-        body: JSON.stringify(userObj),
-        headers: {
-          Accept: 'application/json',
-          'Content-Type': 'application/json'
-        }
-      }
-    );
-
+    const authorizeUser = auth(userDetails);
```

Comment:
We already have Fetch abstract class to handle all request. Try use it to Inherit your auth function